test/e2e_node/doc.go

```diff
@@ -14,4 +14,5 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// e2e_node contains e2e tests specific to the node
 package e2e_node
```

Comment:
(nit) e2e_node doesn't conform to the go package names convention (no underscore - https://blog.golang.org/package-names). You don't need to fix this now though...